diegokim/tdp2-tp0-frontend
# Search screen base and transition

#### What's this PR do?

Introduces the search screen, the transition from/to the main screen

#### Where should the reviewer start?

Test manually and see if it works.

#### How should this be manually tested?

1. Run the project, the title should be Nueva York.
2. Click the menu 3 dots to open the search screen Rosario and Buenos aires are the current options.
3. Click an option and check if the main screen title change to the current city.
